rootbench-scripts/flamegraph.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+BENCHMARKPATTERN="*benchmark*"
+MKDIR=/bin/mkdir
```

Comment:
We should use `MKDIR=$(which mkdir)`. Likewise for the rest.

Reply:
okay! thanks